org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/output/AnalyzeResult.java

```diff
@@ -0,0 +1,69 @@
+package org.metaborg.spoofax.shell.output;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import org.metaborg.core.context.IContext;
+import org.metaborg.core.messages.IMessage;
+import org.metaborg.spoofax.core.stratego.IStrategoCommon;
+import org.metaborg.spoofax.core.unit.ISpoofaxAnalyzeUnit;
+import org.metaborg.spoofax.shell.commands.SpoofaxCommand;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.assistedinject.AssistedInject;
+
+/**
+ * Represents an {@link AnalyzeResult} as returned by the {@link SpoofaxCommand}.
+ * Wraps a {@link ISpoofaxAnalyzeUnit}.
+ */
+public class AnalyzeResult extends AbstractResult<ISpoofaxAnalyzeUnit> {
+
+    /**
+     * Create a {@link AnalyzeResult}.
+     * @param common  the {@link IStrategoCommon} service
+     * @param unit    the wrapped {@link ISpoofaxAnalyzeUnit}
+     */
+    @AssistedInject
+    public AnalyzeResult(IStrategoCommon common,
+                         @Assisted ISpoofaxAnalyzeUnit unit) {
+        super(common, unit);
+    }
+
+    // Duplication here and in TransformResult is intentional since no common ancestor of
+    // ISpoofaxAnalyzeUnit and ISpoofaxTransformUnit exists with these functions.
+    @SuppressWarnings("CPD-START")
```

Comment:
Won't it be neater to suppress these two files from the Maven config? It's less clutter in the code, although it does disable the whole file...